engine/src/main/java/org/hibernate/validator/internal/metadata/aggregated/BeanMetaData.java

```diff
@@ -87,13 +89,22 @@
 	 */
 	Set<MetaConstraint<?>> getMetaConstraints();
```

Comment:
Couldn't we get rid of the 2 existing methods? I think they could go away and we should remove the 2 fields in the impl (no need to store the elements twice).

(I just did a quick check so I might be wrong)